.gitignore copy

```diff
@@ -0,0 +1,34 @@
+# The directory Mix will write compiled artifacts to.
```

Comment:
guessing you just didn't spot that this file had been copied ... 💭 
Removing. ✂️